tests/console/install_packages.pm

```diff
@@ -16,7 +16,9 @@ sub run() {
 
     my $packages = get_var("INSTALL_PACKAGES");
 
-    assert_script_run("zypper -n in -l $packages");
+    assert_script_run("zypper -n in -l perl-solv");
+    assert_script_run("~$username/data/lsmfip --verbose $packages > ~/install_packages.txt");
```

Comment:
from past experiences: avoid ~ for temporary files. It too easily leaks into other tests